### PR TITLE
T2346

### PR DESCRIPTION
**Pull request by secretrobotron**

### CODE REVIEW

File: src/editor/media-editor.js [hunk 1]
```diff
@@ -5,292 +5,299 @@
 define( [ "util/lang", "editor/editor", "util/uri", "text!layouts/media-editor.html" ],
   function( LangUtils, Editor, Uri, EDITOR_LAYOUT ) {
 
-  return function( butter ) {
```
Comment (secretrobotron) [inline src/editor/media-editor.js:8]: Note: the majority of this file's changes is the result of an indentation reduction.

File: src/main.js [hunk 2]
```diff
@@ -13,51 +13,30 @@
...
-            "./dependencies",
-            "./dialogs",
-            "dialog/dialog",
-            "editor/editor",
-            "ui/ui",
-            "util/xhr",
-            "util/lang",
-            "text!default-config.json",
-            "text!layouts/ua-warning.html",
+            "core/eventmanager", "core/logger", "core/config", "core/target", "core/media", "core/page",
```
Comment (jbuck) [inline src/main.js:16]: I don't think this change should be made; what's wrong with the vertical list? Aside from it being long

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @humphd received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/2cbc99ef10fc727/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @humphd received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/5b49c822a2aced4/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/2cbc99ef10fc727/output.txt

Total script time: 2.16 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/5b49c822a2aced4/output.txt

Total script time: 1.03 mins

Server is running on http://pr1016.butyr.org/templates/basic/
Run the Test Suite: http://pr1016.butyr.org/test/

**Pull request merged**